### ISSUE TITLE
Alan wake 2 support

### ISSUE THREAD

**Issue by rm-NoobInCoding** [TomEvin/neat#1]
AW2 is out, but it seems to have different structures compared to the previous versions of northlight engine
Please release the source of the tool so that others can help you to support AW2

**Comment by TomEvin** [comment]
I am working on the AW2 containers, indeed different format.

**Comment by AlekseiBar** [comment]
We are all looking forward to))

**Comment by Narayanna** [comment]
We are waiting, I hope it will happen soon. Thank you.

**Comment by Julz876** [comment]
This is definitely great news!

**Comment by Fodjet** [comment]
Seems it's not that easy to make it work

**Comment by TemeASD** [comment]
Maybe this repository can be of assistance. That tool works, but its kinda PIA to work with. 

https://github.com/amrshaheen61/Alan-Wake-2-RMDTOC-Tool

**Comment by JoelStar89** [comment]
Yes, this tool work, but you can't replace for example audio files with it. Something to unpack and repack correctly the folders could be great... I hope for this!

**Comment by Fodjet** [comment]
> Maybe this repository can be of assistance. That tool works, but its kinda PIA to work with.
> 
> https://github.com/amrshaheen61/Alan-Wake-2-RMDTOC-Tool

I need this tool to extract models from it. I wanna make a mask of Cult Of The Tree but I don't thing I will able to do it from 0

**Comment by TomEvin** [comment]
I am currently translating the game (AW2) to my native language. That is why the original project (for AW1) and it's upgraded version were started, and why I created this project and started it now from scratch [no pun intended] (beside educate myself).
The project is on hold until my translation is not done, I can update this project probably earlier next year.

@TemeASD Thank you the link. I am following scripts and tools created by others for AW2, and what they discovered so far (and what I could find) is currently enough for extracting, repack/create is not really solved as far as I could see, but it is on my to-do list.

Also I created a Poll here: https://github.com/TomEvin/neat/discussions/2

**Comment by TomEvin** [comment]
AW2 support done, tested successfully in PC version.
Neat will be released on October, once Resonant is available and I had chance to check the files.

**Comment by wp760836** [comment]
@TomEvin Control Resonant had been released!!!I am really looking forward to this tool！BTW,It seems like the stucture of files inside base-generi.crmdblob and stream0-generic.rmdblob are different to files in other rmdblob files.   base/stream-XXlanguage.rmdtoc/rmdblob are the same to AW2

**Comment by Chakaberia023** [comment]
please i need this tool
